btc-lsp/test/TestAppM.hs

```diff
@@ -81,6 +81,8 @@ runTestApp env app =
 instance (MonadUnliftIO m) => I.Env (TestAppM 'LndLsp m) where
   getGsEnv =
     asks $ envGrpcServerEnv . testEnvLsp
+  getRpcEnv =
```

Comment:
Maybe you want to use more clean `withElectrs` and `withElectrsT` combinators instead of env reader, similar to how it's done in `withLnd` and `withLndT`

Reply:
yes, good idea